collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_52.xml

```diff
@@ -52,173 +45,320 @@
+                           <item>Sermo dominica 5ª post Pascham petite etc., 173º</item>
+                           <item>Postilla super Cantica fratris Egidii, 176</item>
+                           <item>Soliloquium Richardi</item>
+                        </list>
+                     </quote>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
-                     <supportDesc material="perg">
-                        <support>parchment</support>
+                     <supportDesc material="mixed">
+                        <support>parchment and paper (fols. 174–179, 182–187)</support>
+                        <extent>ii (modern paper) + ii (parchment fragment) + 205 + ii (modern paper) leaves <dimensions unit="mm" type="leaf">
+                              <height>180</height>
+                              <width>130</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Contemporary foliation, ‘1‘–‘45‘ (fols. 1v–43r). Modern foliation doubles several numbers: fols. 5a/5b, 23a/23b, 46a/46b, 67a/67b.</foliation>
+                        <collation>a<hi rend="superscript">two</hi> (fols. iii–iv, attached singletons) | 1–3<hi rend="superscript">12</hi> (fols. 1–34, including 5a/5b, 23a/23b) | 4–6<hi rend="superscript">12</hi> (fols. 35–68, including 46a/46b, 67a/67b) | 7<hi rend="superscript">12</hi> (fols. 69–80), 8–9<hi rend="superscript">10</hi> (fols. 81–100) | 10–15<hi rend="superscript">12</hi> (fols. 101–172) | 16<hi rend="superscript">16</hi> (fols. 173–188) | 17<hi rend="superscript">12–1</hi> (fols. 189–199; 12th cancelled)</collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout writtenLines="50 65" columns="2">Ruled in ink, 2 columns of 50–65 lines</layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="textualisNorthern">Textura by at least two hands (the second from fol. 000).</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="colInit">Space left for initials, not executed.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <binding notBefore="1700" notAfter="1760">
+                        <p>Soranzo binding: brown tanned calf over pulpboard.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1410" notBefore="1400">15th century, beginning</origDate>
+                     <origDate calendar="Gregorian" notAfter="1400" notBefore="1350">14th century, second half</origDate>
+                     <origPlace key="place_1000080">Italian</origPlace>
```

Comment:
Is the origin certain? The German items seem a bit odd in an Italian context perhaps?